src/lib/object-class.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * Returns the object class for any value.
+ *
+ * @param {*} [value] - A value whose object class should be determined.
+ *
+ * @example
+ * const isNull = objectClass(null) === 'null'
+ * const isPOJO = objectClass({}) === 'object'
+ *
+ * @returns {String}
+ */
+const objectClass = (value) => /\[\w+ (\w+)]/.exec(Object.prototype.toString.call(value))[1].toLowerCase()
```

Comment:
Found an [SO post](http://stackoverflow.com/questions/1249531/how-to-get-a-javascript-objects-class) on this, they reference `getClass()` from Java, that is an option.